lib/client.js

```diff
@@ -36,15 +36,15 @@ function* createClient(rabbitmqUrl, options) {
   const busClient = Object.assign(Object.create(EventEmitter.prototype), {
     channel,
     connection,
-    setupQueue,
-    consume,
-    listen,
+    setupQueue: co.wrap(setupQueue),
+    consume: co.wrap(consume),
+    listen: co.wrap(listen),
```

Comment:
Don't understand why you need to do this ?

Reply:
I don't understand either

Reply:
I do this because in projects using async/await, I can't use generators directly. So I have to use Co to wrap it. IMO it's better to do it here because then it is done in only 1 place.
exemple: https://github.com/transcovo/c3po/blob/master/src/util/bus.js#L35